src/http_client.ts

```diff
@@ -130,6 +131,11 @@ export class HttpClient {
         });
     }
 
+    private _informAboutStreamNoRetry() {
+        this.logger.warning('Request body was a stream - retrying will not work, as part of it was already consumed.');
+        this.logger.info('If you want the SDK to handle retries, collect the stream into a buffer before sending it.');
```

Comment:
The first message will be logged once, the second will be logged every time. Is this intended? I would log both as a warning.

Reply:
ad 1] i just checked for that more carefully and turns out that even the warning is being logged repeatedly. is that a bug elsewhere? but no, the intended behavior is that both messages are logged just the first time

ad 2] oops, thanks

Reply:
Oh, sorry. My bad. It's this one. Could you please also send a PR to the log package making an alias (to not make it a breaking change) called `warningOnce`? We've been talking about that multiple times, but seem like we've never done it.

https://github.com/apify/apify-shared-js/blob/c7978d1d93dfa53336504f3023d0be537fdf77a0/packages/log/src/log.ts#L257-L266

Reply:
PR here: https://github.com/apify/apify-shared-js/pull/316 , will make it both a warning then. Is it better to keep it as two separate calls, so both lines are prefixed with the logging info, or should I just throw a `\n` in there?

Reply:
I would vote for renaming the function (as `warningOnce` is exactly what it does) and having `deprecated` as the alias instead.

Reply:
@mvolfik I would just throw an `\n` there, but it's really personal preference.
@B4nan Yeah, makes sense.

Reply:
Ooof, I just realized that the PR was released as version v2.1.0, while all deps of this package depend on 1.*. How should I proceed?

Reply:
Pushed a fix which includes update to the dependency